tools/schemacode/schemacode/validator.py

```diff
@@ -658,6 +664,34 @@ def log_errors(validation_result):
         lgr.warning("The `%s` file was not matched by any regex schema entry.", i)
 
 
+def _query_pseudofile_suffixes(my_schema):
```

Comment:
I think the name is a little unclear. What about `_identify_directory_suffixes`?